### PR TITLE
Fix sciq.

### PR DESCRIPTION
**Pull request by stephenbach**
Still suboptimal that the answer is always last in the multiple choice questions, but that's a pre-existing issue.

### CODE REVIEW

File: promptsource/templates/sciq/templates.yaml [hunk 1]
```diff
@@ -2,28 +2,30 @@ dataset: sciq
...
   15b0a989-84e4-4f1c-8ac1-12dbfa2ff42a: !Template
     answer_choices: null
-    answer_choices_key: distractor1 ||| distractor2 ||| distractor3 ||| correct_answer
+    answer_choices_key: '{{distractor1}} ||| {{distractor2}} ||| {{distractor3}} |||
+      {{correct_answer}}'
     id: 15b0a989-84e4-4f1c-8ac1-12dbfa2ff42a
-    jinja: "Q: {{question}}\n\n\n Choices:\n\n- {{ distractor1 }}\n\n- {{ distractor2\
-      \ }}\n\n- {{ distractor3 }}\n\n- {{ correct_answer }}\n\nA:|||{{correct_answer}}"
+    jinja: "Q: {{question}}\n\n\n Choices:\n\n- {{ answer_choices[0] }}\n\n- {{ answer_choices[1]\
+      \ }}\n\n- {{ answer_choices[2] }}\n\n- {{ answer_choices[3] }}\n\nA:|||{{answer_choices[3]}}"
```
Comment (VictorSanh) [inline promptsource/templates/sciq/templates.yaml:28]: only thing i am worried about is that the order is deterministic in the sense that the last option is always the right one

File: promptsource/templates/sciq/templates.yaml [hunk 2]
```diff
@@ -48,11 +51,12 @@ templates:
...
     answer_choices: null
-    answer_choices_key: distractor1 ||| distractor2 ||| distractor3 ||| correct_answer
+    answer_choices_key: '{{distractor1}} ||| {{distractor2}} ||| {{distractor3}} |||
+      {{correct_answer}}'
     id: 63c22e8a-7029-4ce3-bd26-6ca6a1541563
     jinja: "Answer the following question given this paragraph: \n\n{{support}}\n\n\
-      \nQ: {{question}}\n\n Choices:\n\n- {{ distractor1 }}\n\n- {{ distractor2 }}\n\
-      \n- {{ distractor3 }}\n\n- {{ correct_answer }}\n\nA:|||{{correct_answer}}\n\
+      \nQ: {{question}}\n\n Choices:\n\n- {{ answer_choices[0] }}\n\n- {{ answer_choices[1]\
+      \ }}\n\n- {{ answer_choices[2] }}\n\n- {{ answer_choices[3] }}\n\nA:|||{{answer_choices[3]}}\n\
```
Comment (VictorSanh) [inline promptsource/templates/sciq/templates.yaml:59]: same comment

File: promptsource/templates/sciq/templates.yaml [hunk 3]
```diff
@@ -33,12 +35,13 @@ templates:
...
+    answer_choices_key: '{{distractor1}} ||| {{distractor2}} ||| {{distractor3}} |||
+      {{correct_answer}}'
     id: 368e29fb-506d-4a4e-ac33-0af8d6e1729b
     jinja: "Q: {{question}}\n\n\nRead this paragraph and choose the correct option\
-      \ from the provided answers:\n\n{{support}}\n\n Choices:\n\n- {{ distractor1\
-      \ }}\n\n- {{ distractor2 }}\n\n- {{ distractor3 }}\n\n- {{ correct_answer }}\n\
-      \n\nA:|||{{correct_answer}}\n"
+      \ from the provided answers:\n\n{{support}}\n\n Choices:\n\n- {{ answer_choices[0]\
+      \ }}\n\n- {{ answer_choices[1] }}\n\n- {{ answer_choices[2] }}\n\n- {{ answer_choices[3]\
+      \ }}\n\n\nA:|||{{answer_choices[3]}}\n"
```
Comment (VictorSanh) [inline promptsource/templates/sciq/templates.yaml:44]: same comment

**Review by VictorSanh — APPROVED** [review]
small comment on the order which is related to the discussion around whether the original task are too easy. thoughts?

File: promptsource/templates/sciq/templates.yaml (hunk 1 above)
Comment (stephenbach) [reply promptsource/templates/sciq/templates.yaml:28]: Yeah, it's annoying. I could add some code to randomize the order by calling choice on all permutations. Might not get to it until later this afternoon, if you want to unblock @craffel with the bug fixes. I could do a separate PR.

Comment (VictorSanh) [reply promptsource/templates/sciq/templates.yaml:28]: ok let's do separate PR

**Comment by VictorSanh** [comment]
merging now

**Pull request merged**